src/styles/main.scss

```diff
@@ -61,6 +57,10 @@
  * @import "tailwindcss/utilities";
  */
 @tailwind utilities;
+@import "utilities/_ratio";
+@import "utilities/_object-fit";
+@import "utilities/_transitions";
+@import "utilities/_animations";
 @import "base/tailwind-custom-utilities";
```

Comment:
`@import "base/tailwind-custom-utilities";` is not needed anymore or should at least be in "utilities" as "_custom-utilities"